src/read.jl

```diff
@@ -38,8 +38,10 @@ function read_case(filepath)
     case["gen_ramp30"] = convert(Array{Float64,1}, plant.ramp_30)
 
     # Generator costs
-    gencost = DataFrames.DataFrame(CSV.File(joinpath(filepath, "gencost.csv")))
-    case["gencost"] = convert(Matrix{Float64}, gencost)
+    case["gencost"] = DataFrames.DataFrame(CSV.File(joinpath(filepath, "gencost.csv")))
+    case["gencost_orig"] = DataFrames.DataFrame(
+        CSV.File(joinpath(filepath, "gencost_orig.csv"))
+    )
```

Comment:
Leave a mark here: if we use `gencost_before` and `gencost_after` as @rouille suggested, we will need to modify these accordingly.